en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md

```diff
@@ -60,79 +54,79 @@ Now follow the steps below to add resources to the API.
-       - **provisionedThroughput** : Represents the provisioned throughput setting for a specified table or index.
-        
+
+        - **attributeDefinitions** : A list of attributes that describe the key schema for the table and indexes. If you are adding a new global secondary index to the table, AttributeDefinitions should include the key element(s) of the new index. 
+        - **tableName** : The name of the table to create. 
+        - **keySchema** : Specifies the attributes that make up the primary key for a table or an index. The attributes in keySchema must also be defined in attributeDefinitions.
+        - **localSecondaryIndexes** : One or more local secondary indexes (the maximum is five) to be created on the table. Each index is scoped to a given partition key value. There is a 10 GB size limit per partition key value. Alternately, the size of a local secondary index is unconstrained. 
+        - **provisionedThroughput** : Represents the provisioned throughput setting for a specified table or index.
+
         While invoking the API, the above five parameter values come as a user inputs.
-        
-        <img src="{{base_path}}/assets/img/integrate/connectors/amazon-dynamodb-drag-and-drop-create-table-parameters.jpg" title="Drag and drop create table operation" width="500" alt="Drag and drop create table operation"/> 
-    
-    3. To get the input values in to the API we can use the [property mediator]({{base_path}}/reference/mediators/property-mediator). Navigate into the **Palette** pane and select the graphical mediators icons listed under the **Mediators** section. Then drag and drop the `Property` mediators into the Design pane as shown below.
-    
-        <img src="{{base_path}}/assets/img/integrate/connectors/amazon-dynamodb-drag-and-drop-property-mediator.png" title="Add property mediators" width="800" alt="Add property mediators"/>
+
+        <img src="{{base_path}}/assets/img/integrate/connectors/amazon-dynamodb/amazon-dynamodb-drag-and-drop-create-table-parameters.png" title="Drag and drop create table operation" width="800" alt="Drag and drop create table operation"/> 
+
+    3. To get the input values in to the API we can use the [property mediator]({{base_path}}/reference/mediators/property-mediator). Navigate into the **Mediators** pane and select the `Property` mediator.
+
+        <img src="{{base_path}}/assets/img/integrate/connectors/amazon-dynamodb/amazon-dynamodb-drag-and-drop-property-mediator.png" title="Add property mediators" width="800" alt="Add property mediators"/>
 
         The parameters available for configuring the Property mediator are as follows:
     
         > **Note**: That the properties should be add to the pallet before create the operation.
```

Comment:
`That the properties should be add to the pallet before create the operation.` This sentence is not complete and grammatically correct. 
It should be
`**Note**: The properties should be added to the pallet before creating the operation.`